src/main/java/org/kiwiproject/migrations/mongo/MongoMigrationConfiguration.java

```diff
@@ -4,4 +4,8 @@
     String getMigrationPackage(T config);
     String getMongoUri(T config);
     String getDatabaseName(T config);
+
+    default boolean disableTransactions(T config) {
```

Comment:
Are you going to rename this to `shouldDisableTransactions` like we discussed offline?

Reply:
Yes